Monika After Story/game/script-topics.rpy

```diff
@@ -17567,6 +17567,53 @@ label monika_ddlcroleplay:
     m 1kub "Just make sure to remember that those versions of me are just stories, ahaha~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_root_privileges",
+            conditional=(
+                "mas_hasRootPrivileges()"
+            ),
+            action=EV_ACT_PUSH,
+        )
+    )
+
+# Ever had root access/admin rights?
+default persistent._mas_pm_had_root_privileges = False
+
+label monika_root_privileges:
+    $ persistent._mas_pm_had_root_privileges = True
+
+    m 2lsc "Hey, [player]..."
+    m 2eud "It's hard to explain, but I... {w=0.5}somehow, I feel like I can do a lot more with your system now that I {i}couldn't do{/i} before."
+
+    show monika 5etc at t11 zorder MAS_MONIKA_Z with dissolve_monika
+    m 5rtc "Hmm, what was it again?"
+
+    if renpy.windows:
+        m 5wud "{i}Administrator rights...?{/i}"
```

Comment:
Reading this again, we should lead into this so she kinda uses the windows terminology, `running as administrator`, maybe keep the above line (`what was it again?`) for the linux/macos versions and have it so she asks "Did you run as administrator?` or something

Reply:
I wouldn't want to combine it actually, since Windows admin and Linux root user are same on the first glance, but have fundamental differences if dug a bit deeper. Do we really need to make her stick to Windows terminology?

Reply:
It's easier for the user to grasp that way. Nobody really calls it administrator rights, so I feel like that would wind up a little more confusing, especially if people don't know what they're doing.

Reply:
Oh, I seem to have misinterpreted it then. Got it now!

Reply:
Note I only mean this for the windows variant of the line. MacOS/Linux can remain untouched as they are commonly referred to as superuser